pkgs/by-name/ze/zerocool_ttf/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "zerocool_ttf";
```

Comment:
The conversation at https://github.com/NixOS/nixpkgs/pull/416336/files#r2144205294 might be about the same topic.

The term "_ttf" heavily relies on the current file name. If the upstream adds other font types to its assets, this naming will quickly become unsuitable. It might be better to generalize it, for example, by replacing "ttf" with "font."

While the SDL2_ttf package exists and its URL clarifies TTF (https://github.com/NixOS/nixpkgs/blob/644eba86f28cdd834afc55b55dee9cdbca410e23/pkgs/by-name/sd/SDL_ttf/package.nix#L47), this package's upstream only provides the URL `zero-cool-font`.